lib/elixir/test/elixir/kernel/import_test.exs

```diff
@@ -20,8 +20,10 @@ defmodule Kernel.ImportTest do
   end
 
   test "import except one" do
-    import :lists, except: [each: 2]
+    import :lists, except: [duplicate: 2]
```

Comment:
Not entirely sure about this one.  I went back and forth between adding `duplicate` and replacing `each` with `duplicate`.  The problem with adding it is that it no longer really tests "import except one" but rather "import except two".  On the other hand, it seemed like maybe the intention was to test it out with a non-existent function in the except clause, in which case we don't want to lose `each`.  So let me know what you think.